src/components/BasketCard/index.js

```diff
@@ -0,0 +1,52 @@
+import React , { useState,useEffect }from 'react'
+import { selectAppLoading } from '../../store/appState/selectors'
+import { useSelector } from 'react-redux'
+import './index.css'
+
+import { Button } from 'react-bootstrap'
+export default function BasketCard(props) {
+    const isLoading = useSelector(selectAppLoading);
+    
+    let {id,name,description,imageUrl,price,onClickAddItemToCard, onClickRemoveItemCard, quantity,productId} = props
```

Comment:
spaces after `,` and on object start-end:
```
let { id, name, description }
```

Reply:
also, why `let` and not `const`, you never reassign any of them. Well you do `price` but that can be easily solved in another way